### PR TITLE
Update examples.

### PR DESCRIPTION
**Pull request by Kelvin-M**
Update CsrHost for local values with .svc.cluster.local to avoid potential issue with network resolution like "Invalid toplevel subdomain" issue in substra-backend grpc client.

Companion PR : https://github.com/SubstraFoundation/substra-backend/pull/385

### CODE REVIEW

**Comment by Kelvin-M** [comment]
@ClementGautier @AurelienGasser  @inalgnu https://github.com/SubstraFoundation/substra-backend/pull/385 can not be merged if this one is not reviewed :)

File: examples/2-orgs-policy-any-no-ca-same-cc/values/orderer.yaml [hunk 1]
```diff
@@ -35,7 +35,7 @@ enrollments:
   creds:
   - { name: admin, secret: adminpwd, options: "--id.attrs admin=true:ecert"}
   - { name: user, secret: pwd, options: "--id.type orderer"}
-  csrHost: network-orderer-hlf-ord.orderer
+  csrHost: "network-orderer-hlf-ord.orderer,network-orderer-hlf-ord.orderer.svc.cluster.local"
```
Comment (ClementGautier) [inline examples/2-orgs-policy-any-no-ca-same-cc/values/orderer.yaml:38]: image

Je serais pour garder qu'un seul des deux, network-orderer-hlf-ord.orderer.svc.cluster.local dans ce cas pour régler le soucis remonté sur https://github.com/SubstraFoundation/substra-backend/pull/385.

Comment (Kelvin-M) [reply examples/2-orgs-policy-any-no-ca-same-cc/values/orderer.yaml:38]: It means that we need to update all host values from examples too ! Can you confirm it ?

Comment (ClementGautier) [reply examples/2-orgs-policy-any-no-ca-same-cc/values/orderer.yaml:38]: I mean, yes. It seems weird to have a something called A at one place and B at the other pointing to the same service. `sed -i "s/network-orderer-hlf-ord.orderer/network-orderer-hlf-ord.orderer.svc.cluster.local/g"` is the way

**Review by ClementGautier — APPROVED** [review]

**Pull request merged**